episodes/filters-and-thresholding.md

```diff
@@ -310,44 +295,35 @@ change its colormap from 'green' to 'gray' in the layer controls.
 
 ![](fig/nuclei-gray.png){alt="Nuclei image with gray colormap" width='40%'}
 
-With the new plugins installed, you should see many new options under `Tools` in 
+With the new plugin installed, you should see many new options under `Layers` in 
 the top menu-bar of Napari. You can find out more about these options using the 
-plugin's documentation which was linked in the [previous section
-](#adding-plugins-to-help-with-segmentation).
-
-For now, we are interested in the options under:  
-`Tools > Filtering / noise removal`
+[plugin's documentation](https://github.com/guiwitz/napari-skimage/blob/main/README.md).
 
-Specifically, the two options labelled 'gaussian' - you should see one ending in 
-'n-SimpleITK' and another ending 'scikit-image, nsbatwm'. These are two slightly 
-different implementations of a _gaussian blur_, one from the 
-`napari-simpleitk-image-processing` plugin and the other from the 
-`napari-segment-blobs-and-things-with-membranes` plugin. Both should work 
-perfectly well for our image, but let's use the one ending with 'scikit-image, 
-nsbatwm' for now.
+For now, we are interested in the _gaussian blur_ under:  
+`Layers > Filter > Filtering > Gaussian filter (napari-skimage)`
 
 If you click on this option, you should see a new panel appear on the right side 
 of Napari:
 
 ![](fig/gaussian-options.png){alt="Screenshot of settings for gaussian blur in 
 Napari" width='40%'}
 
-Make sure you have 'nuclei(data)' selected on the 'image' row, then click run:
+Make sure you have 'nuclei' selected on the 'image' row, then click 
+'Apply Gaussian Filter':
 
 ![](fig/nuclei-blurred-1.png){alt="Nuclei image after gaussian blur with sigma 
 of 1" width='40%'}
 
-You should see a new image appear in the layer list called 'Result of 
-gaussian_blur', which is a slightly blurred version of the original nuclei 
-image.
+You should see a new image appear in the layer list called 
+'nuclei_gaussian_σ=1.0', which is a slightly blurred version of the 
+original nuclei image.
 
 Try increasing the 'sigma' value to three and clicking run again:
 
 ![](fig/nuclei-blurred-3.png){alt="Nuclei image after gaussian blur with sigma 
 of 3" width='40%'}
 
-You should see that the 'Result of gaussian_blur' layer is updated to show a 
-much more heavily blurred version of the original nuclei image.
+You should see a new 'nuclei_gaussian_σ=3.0' layer that is much more heavily blurred.
```

Comment:
I set sigma to 3.0, but got sigma = 3.00....18. Weird.
<img width="517" height="199" alt="Screenshot from 2025-11-03 11-33-21" src="https://github.com/user-attachments/assets/e391bd2f-cc9d-4524-8b87-ab0a21c3375f" />